### PR TITLE
Adding translation for Portuguese-Portugal and Portuguese-Brazil

### PR DESCRIPTION
**Pull request by thaleseuflauzino**

### CODE REVIEW

File: src/locales/br/translation.json [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+  },
+  "sidebar": {
+    "catalogue": "Catálogo",
+    "downloads": "Downloads",
+    "settings": "Ajustes",
+    "my_library": "Biblioteca",
+    "downloading_metadata": "{{title}} (Baixando metadados…)",
+    "paused": "{{title}} (Pausado)",
+    "downloading": "{{title}} ({{percentage}} - Baixando…)",
+    "filter": "Buscar",
```
Comment (agnjunio) [inline src/locales/br/translation.json:19]:
```suggestion
    "filter": "Filtrar bibilioteca",
```

File: src/locales/br/translation.json [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+  "home": {
+    "featured": "Destaques",
+    "trending": "Populares",
+    "surprise_me": "Surpreenda-me",
+    "no_results": "Nenhum resultado encontrado"
+  },
+  "sidebar": {
+    "catalogue": "Catálogo",
+    "downloads": "Downloads",
+    "settings": "Ajustes",
```
Comment (agnjunio) [inline src/locales/br/translation.json:14]:
```suggestion
    "settings": "Configurações",
```

File: src/locales/br/translation.json [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+    "queued": "{{title}} (Na fila)",
+    "game_has_no_executable": "Jogo não possui executável selecionado",
+    "sign_in": "Login"
+  },
+  "header": {
+    "search": "Buscar jogos",
+    "catalogue": "Catálogo",
+    "downloads": "Downloads",
+    "search_results": "Resultados da busca",
+    "settings": "Ajustes",
```
Comment (agnjunio) [inline src/locales/br/translation.json:30]:
```suggestion
    "settings": "Configurações",
```

File: src/locales/br/translation.json [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+    "danger_zone_section_title": "Zona de perigo",
+    "danger_zone_section_description": "Remova o jogo da sua biblioteca ou os arquivos que foram baixados pelo Hydra",
+    "download_in_progress": "Download em andamento",
+    "download_paused": "Download pausado",
+    "last_downloaded_option": "Última opção baixada",
+    "create_shortcut_success": "Atalho criado com sucesso",
+    "create_shortcut_error": "Erro ao criar atalho"
+  },
+  "activation": {
+    "title": "Ativação",
```
Comment (agnjunio) [inline src/locales/br/translation.json:115]:
```suggestion
    "title": "Ativar Hydra",
```

File: src/locales/br/translation.json [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+    "queued_downloads": "Na fila",
+    "downloads_completed": "Completo",
+    "queued": "Na fila",
+    "no_downloads_title": "Nada por aqui…",
+    "no_downloads_description": "Você ainda não baixou nada pelo Hydra, mas nunca é tarde para começar.",
+    "checking_files": "Verificando arquivos…"
+  },
+  "settings": {
+    "downloads_path": "Diretório dos downloads",
+    "change": "Explorar...",
```
Comment (agnjunio) [inline src/locales/br/translation.json:149]:
```suggestion
    "change": "Alterar",
```
English says `Update` but I think this is a file picked so `Explorar...` is good.

**Review by agnjunio — COMMENTED** [review]
Some notes from a Brazilian guy 😅

**Comment by zamitto** [comment]
The ISO-639-1 standard does not differ European Portuguese and Brazilian Portuguese. 

Currently Hydra uses a library that uses ISO-639-1 to show the language options in the settings-general.tsx file. 
Running your code you can see that the "br" export results in the library showing the "brezhoneg" option.

image

It would be awesome if you could find a library that supports this format.
Another possibility would be to map these laguanges manually. But I'm not sure that would be good.

**Comment by thaleseuflauzino** [comment]
> Some notes from a Brazilian guy 😅

thanks

**Comment by thaleseuflauzino** [comment]
> The ISO-639-1 standard does not differ European Portuguese and Brazilian Portuguese.
> 
> Currently Hydra uses a library that uses ISO-639-1 to show the language options in the settings-general.tsx file. Running your code you can see that the "br" export results in the library showing the "brezhoneg" option.
> 
> image
> 
> It would be awesome if you could find a library that supports this format. Another possibility would be to map these laguanges manually. But I'm not sure that would be good.

I resolved the issue by using the @cospired/i18n-iso-languages package from Yarn and manually distinguishing between pt-BR and pt-PT. This ensures that the application correctly recognizes the different Portuguese language variants in the interface.

**Comment by thaleseuflauzino** [comment]
@zamitto solved

**Comment by thaleseuflauzino** [comment]
@zamitto i dont know whats the conflict, i believe can be the delete from `src/locales/pt/translation.json`, please check

**Comment by thaleseuflauzino** [comment]
@zamitto done

**Pull request merged**